src/readme.txt

```diff
@@ -54,25 +53,30 @@ The 'admin_*' hooks are designed for wp-admin area or so called Dashboard area,
 
 == Changelog ==
 
+= 1.5 =
+* WP-CLI skeleton
```

Comment:
WP-CLI?? dose does not matter if it was generated using wp-cli. maybe it's better to just say something like "upgrade repo structure to match recommended structure."